examples/complexNumbers/src/complexNumber.js

```diff
@@ -0,0 +1,83 @@
+class ComplexNumber {
+
+  constructor(real, imag) {
+    if(typeof real == 'number' && typeof imag == 'number')
+    {
+      this.real = real;
+      this.imag = imag;
+    }
+    else
+    {
+      this.real = 0;
+      this.imag = 0;
+    }
```

Comment:
Raise exception if args are not OK. This way you would just mask error. It's better to 😱 as soon as something is fishy.